docs/docs/Configuration.md

```diff
@@ -8,7 +8,58 @@ index: 5
 Fantomas ships with a series of format options.
 These can be stored in an [.editorconfig](https://editorconfig.org/) file and will be picked up automatically by the commandline tool.
 
-A default .editorconfig file would look like
+<!-- 
+    This index can be auto-generated with VS Code's Markdown All in One extension.
+    The ToC will be updated-on-save, or can be generated on command by using
+    Ctrl-Shift-P: "Create table of contents". 
+    More info: https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one#table-of-contents 
+-->
+## Index<!-- omit in toc -->
+- [Configuration](#configuration)
+- [Default `.editorconfig` settings](#default-editorconfig-settings)
```

Comment:
It should be `#Default-settings` please verify all these links locally.

Reply:
Actually, it's `#default-editorconfig-settings`, I did verify, see: https://github.com/fsprojects/fantomas/blob/b11b645a997a0baa2dea51a7c1aba570ce9dceca/docs/docs/Configuration.md#default-editorconfig-settings